actionpack/test/dispatch/request_test.rb

```diff
@@ -1061,7 +1061,7 @@ class RequestParameterFilter < BaseRequestTest
 
       filter_words << 'blah'
       filter_words << lambda { |key, value|
-        value.reverse! if key =~ /bargain/
```

Comment:
I see. So this means that after this PR applications that have filters that mutate the value will have to change the code?